src/clj_sqs_extended/aws/sqs.clj

```diff
@@ -155,29 +169,40 @@
   (->> (DeleteMessageRequest. queue-url (:receiptHandle message))
        (.deleteMessage sqs-client)))
 
+(defn- get-serdes-format-attribute
+  [message]
+  (when message
+    (let [attributes (.getMessageAttributes message)
+          format (get attributes clj-sqs-ext-format-attribute)]
+      (keyword (.getStringValue format)))))
+
+(defn- extract-relevant-keys
```

Comment:
👍 for refactoring this out. one improvement is to name it more descriptively, e.g. `extract-relevant-keys-from-message`